sgf-viewer/src/components/NavBar.svelte

```diff
@@ -1,185 +1,85 @@
+    // modifications to make it work with Svelte's dynamic element rendering.
+    // (The other solution using HTML attributes given in the SO answer, though
+    // it's simpler, breaks the links' actual href functionality.)
+    $: navLinks = navbarSupportedContent?.querySelectorAll('.nav-item:not(.dropdown)');
+    $: bootstrapCollapse = navbarSupportedContent ? Collapse.getOrCreateInstance(navbarSupportedContent, {toggle: false}) : undefined;
+    $: navLinks?.forEach((link) => {
+           link.addEventListener('click', () => { bootstrapCollapse.toggle() });
+    })
 </script>
 
-<svelte:window bind:innerWidth />
-<!-- Empty anchor target. Named for link backwards compatibility. -->
-<div id="contents" />
-<div class="nav-bar-container">
-    {#if innerWidth > menuNavigationWidth}
-        <div class="nav-bar" bind:this={navBarElem}>
-            <!-- <div class="icons flex-grow-symmetric">
-                <a
-                    class="icon-link"
-                    on:click={() => {
-                        currentPath = "";
-                        history.pushState({}, "", currentPath);
-                    }}
-                    href="#"
-                >
-                    <TiHome />
-                </a>
-            </div> -->
-            <div class="pages">
-                {#each pagesPaths as page}
-                    <NavButtons
-                        label={pages[page]["title"]}
-                        tabsId="tabs"
-                        selected={page === currentPath}
-                        href="#contents"
-                        onClick={() => {
-                            currentPath = page;
-                            history.pushState({}, "", currentPath + "#contents");
-                        }}
-                    />
-                {/each}
-            </div>
-            <!-- <div class="flex-grow-symmetric" /> -->
-        </div>
-    {:else}
-        <div class="menu-bar" bind:this={navBarElem}>
-            <!-- <div class="flex-grow-symmetric" /> -->
-            <!-- <p class="menu-selected">{pages[currentPath]["title"]}</p> -->
-            <!-- <div class="menu-burger flex-grow-symmetric">
-                <div >
-                    <a on:click={() => menuOpen = !menuOpen}>
-                        <IoIosMenu />
-                    </a>
-                </div>
-            </div> -->
-            <!-- {#if menuOpen} -->
-            <select
-                bind:value={currentPath}
-                on:change={() => {
-                    history.pushState({}, "", currentPath);
-                    location.replace("#contents");
-                }}
-            >
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container-fluid">
+        <a class="navbar-brand" href="https://far.ai">
+            <img src="/images/far-logo.svg" alt="FAR logo" width="24" height="24" class="d-inline-block align-text-top">
```

Comment:
Our current [branding guidelines](https://docs.google.com/document/d/1RYEhcJwhDXe4nu0bhqukuh_LPbHWbmYcVK38mC4BnjY/edit) indicate we should use this lock-up https://drive.google.com/file/d/1xGbrKwUKECkMVVshActnInL1ZjxqLOrl/view?usp=sharing (you can just import that and omit the "FAR" text after it -- this will ensure we use the right font / kerning / etc for the FAR wordmark)